### DISCUSSION TITLE
Richiesta convalidazione assenza,ritardo,uscita facoltativa

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#165] [Richieste di aiuto] [answered]
Buongiorno ,sarebbe possibile lasciare il libero arbitrio alla segreteria/amministratore scolastico di attivare o meno la convalidazione dell'assenza ,uscita ,ritardo da parte dei docenti?

**Comment by trinko** [comment, marked as answer]
Tieni presente che già ora, se ignori le convalide non hai conseguenze degne di nota. L'assenza viene comunque considerata giustificata.

Comunque, la segno come richieste per eventuali modifiche future.
